### PR TITLE
Remove scipy and skimage dependencies

### PR DESCRIPTION
**Pull request by jgersak**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,6 +1,5 @@
 lightgbm>=2.0.11
 numpy>=1.13.3
-scikit-image>=0.13.0
-scipy>=0.19.1
+opencv-python~=4.7.0.68
```
Comment (devisperessutti) [inline requirements.txt:3]: I think it would be best to use `opencv-python-headless` to allow compilation on containers

**Review by devisperessutti — CHANGES_REQUESTED** [review]
looks good, minor comment about which package to use

File: requirements.txt (hunk 1 above)
Comment (jgersak) [reply requirements.txt]: updated

File: s2cloudless/utils.py [hunk 2]
```diff
@@ -152,3 +153,12 @@ def download_bands_and_valid_data_mask(
         mask.append(data[..., -1])
 
     return np.array(bands, dtype=np.float32), np.array(mask, dtype=bool)
+
+
+def cv2_disk(radius: int) -> np.ndarray:
+    """
+    Recreates the disk structural element from skimage.morphology using OpenCV.
+    """
```
Comment (zigaLuksic) [inline s2cloudless/utils.py:161]:
```suggestion
    """Recreates the disk structural element from skimage.morphology using OpenCV."""
```

**Review by zigaLuksic — APPROVED** [review]

**Review by devisperessutti — APPROVED** [review]

**Pull request merged**